src/client.ts

```diff
@@ -675,10 +775,20 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
     overrideUserId?: string,
     overrideAttributes?: optimizely.UserAttributes
   ): string | null {
+    if (!this._client) {
+      logger.warn(
+        'Unable to get feature variable string from feature "%s" because Optimizely client failed to initialize.',
+        feature
+      );
+      return null;
+    }
+
     const user = this.getUserContextWithOverrides(overrideUserId, overrideAttributes);
     if (user.id === null) {
+      logger.warn('Unable to get feature variable string from feature "%s" because User ID is not set', feature);
```

Comment:
I dont know when would this case happen but i will suggest to convert it to info. We might not want existing users to start complaining about new warnings on old use cases.